### PR TITLE
docs(sessions): 📝 add learning objectives to unit testing session

### PR DESCRIPTION
**Pull request by signekb**
### Description

This PR includes minor edits to the learning objectives of the `testing` session.

**Reviewers should focus on the objectives (L16-24):** 
- Are we missing something in these objectives that is essential to this session? 
- Does the order make sense?
- Are there bullet points that should be removed?

### CODE REVIEW

**Review by rmj822 — APPROVED** [review]
Looking great!

File: sessions/testing.qmd [hunk 1]
```diff
@@ -15,18 +15,21 @@
 
 **Session objectives**:
 
+-   Gain an understanding of the usefulness and importance of 
+    formal testing
```
Comment (lwjohnst86) [inline sessions/testing.qmd:19]: Nice first start! As I mentioned to @rmj822 in #15, you want the objective to be specific and just as much on the "why/what" as the how.

So in this case, an edit would be:

```suggestion
-   Explain what "testing" means when developing packages, describe how they are structured, and construct simple tests of functions. 
```

In this case, we're saying that learners will remember/recall something, gain understanding on what tests means, and to create/apply this knowledge as a skill to the problem.

**Review by lwjohnst86 — DISMISSED** [review]

**Review by lwjohnst86 — APPROVED** [review]
Great! I reviewed and edited the objectives. I will merge in, but you can look this PR over to learn from my edits 🎉

**Comment by signekb** [comment]
Amazing. Thanks @lwjohnst86 !

**Pull request merged**